examples/bri-3/src/bri/vsm/capabilites/handleWorkstepFailuresEvents/workstepExecutionFailures.event.ts

```diff
@@ -0,0 +1,3 @@
+export class WorkstepExecutionFailuresEvent {
+  constructor(public readonly id: string, public readonly err: string) {}
```

Comment:
add here all relevant data necessary to log the error and construct a bpi message that will be sent to the initiator. No need to read the db again in the event handler. Please look at the other comment as well.